exotic.py

```diff
@@ -541,23 +541,36 @@ def plate_solution(fits_file, saveDirectory):
         time.sleep(5)
 
 #Checks if comparison star coordinates don't point to variable stars
-def variableStarCheck(comparisonStarFile):
+def variableStarCheck(comparisonStarFile, compStarList):
     #Open WCS file and convert comparison star's coordinates into normal coordinates
     starList = fits.open(comparisonStarFile)
     wcsCoords = WCS.WCS(starList[0].header)
     starCoords = wcsCoords.all_pix2world()
+    raList = starCoords[0]
+    decList = starCoords[1]
+    validCoordList = []
 
-    #Use normalized Coordinates to search on SIMBAD to check if that star is variable
-    try:
-        resultTable = Simbad.query_region(coord.SkyCoord(starCoords[0], starCoords[1], unit = (u.deg, u.deg))
-    except TypeError:
-        print("Error: Invalid star coordinates. Aborting process")
-
-    #Check if star ID/name explicitly has V* in it
-    starName = resultTable['MAIN_ID'][0].decode("utf-8")
-    if "V*" in starName:
-        return False
+    #For every comparison star, convert into WCS coordinates and check if it's in WCS file before querying
+    for compStar in compStarList:
+        compStarCoords = compStar.all_pix2world()
+        if compStarCoords[0] in raList and compStarCoords[1] in decList:
+            validCoordList.append((compStarCoords[0], compStarCoords[1]))
 
+    #Use normalized Coordinates to search on SIMBAD to check if that star is variable
+    #Throws an error if invalid coordinates are inputted
+    for counter in range(0, len(validCoordList)):
+        try:
+            resultTable = Simbad.query_region(coord.SkyCoord(validCoordList[counter][0], validCoordList[counter][1], unit = (u.deg, u.deg)))
+        except TypeError:
+            print("Error: Invalid star coordinates. Checking next star... ")
+
```

Comment:
I think you're missing a `continue` here- otherwise, the rest of the `for` loop will continue to run and throw you an error.

Reply:
I just added that per your suggestion; I didn't realize that was a possibility the first time through

Reply:
Still not seeing the `continue` command here.

Reply:
Added to updated version of code not yet pushed (both of us were present to see me add this to the code)